### ISSUE TITLE
Feature request: add `#[tabled(crate = ...)]` to allow re-exporting the library

### ISSUE THREAD

**Issue by unmaykr-aftermath** [zhiburt/tabled#393]
# Problem 
I've run into the following issue while developing a `macro_rules!` macro that auto-implements `Tabled` for a struct:
![image](https://github.com/zhiburt/tabled/assets/98741738/ff699fbd-443b-410e-ae7a-590c55b71a88)
`cargo clippy` output:
```
 1  error[E0433]: failed to resolve: could not find `tabled` in the list of imported crates
```

This happens because there are lines like this in the `derive` implementation: https://github.com/zhiburt/tabled/blob/master/tabled_derive/src/lib.rs#L48

Hence, crates that use my macro will be forced to import both my crate and the `tabled` crate, which is not great ergonomics.

# Request
Add a `#[tabled(crate = ...)]` that takes as argument the path of the `tabled` crate. This will allow crates to re-export `tabled` and let users use their macros without explicitly importing `tabled`.

Basically the same as serde's `#[serde(crate = "...")]` [attribute](https://serde.rs/container-attrs.html#crate).

I have some experience implementing macros like this and can do so for `tabled` if the community/maintainers agree with the addition.

**Comment by zhiburt** [comment]
Hi there

Interesting issue.

If you wanna try you can do it;
But I can do it myself if you wish.

________________

I guess you'll need to update the struct and use it in the trait emit.

https://github.com/zhiburt/tabled/blob/fa06ecfb180649fef55141cae8dfd8db11a8a2c5/tabled_derive/src/attributes.rs#L77

Take care

**Comment by unmaykr-aftermath** [comment]
> Hi there
> 
> Interesting issue.
> 
> If you wanna try you can do it; But I can do it myself if you wish, let me know if so.
> 
> I guess you'll need to update the struct and use it in the trait emit.
> 
> https://github.com/zhiburt/tabled/blob/fa06ecfb180649fef55141cae8dfd8db11a8a2c5/tabled_derive/src/attributes.rs#L77
> 
> Take care

@zhiburt If you wanna have a stab at it first, I think I'll only have time next week for it. Thanks for the tip btw!
